tests/models/cos/test_cos_model.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
```

Comment:
Is it worth adding a test for the pv_nwp model? Sorry if i missed it

Reply:
It's not but it's very simple (basically it just loads the serialized model), which the test on the script also does.
Happy to add one if you think it's worth it.

Reply:
I thinks it would be worth it - thanks

Reply:
Added in 94ce3f50f0c60262b8b66274a296c015160e77dc